Sources/MapboxNavigation/LaneView.swift

```diff
@@ -401,7 +401,7 @@ open class LaneView: UIView {
         self.init(frame: LaneView.defaultFrame)
         backgroundColor = .clear
         self.indications = indications
-        maneuverDirection = direction ?? ManeuverDirection(rawValue: indications.description)
+        maneuverDirection = direction
```

Comment:
It’s probably still reasonable to keep this fallback for when the route doesn’t have the new active direction information.